multi-int-store/src/main/java/com/connexta/multiintstore/common/StorageManager.java

```diff
@@ -66,10 +63,12 @@ private void handleCallback(MetadataCallback callback, String ingestId) {
     LOGGER.info("Metadata");
     //  TODO :: Check Markings
 
+    String contents = null;
```

Comment:
✏️ Why was this moved out?

Reply:
I was originally doing something with this but didn't change this back after the refactor. Good catch